iosApp/Modules/Helpers/AudioPlayer/PlayButton.swift

```diff
@@ -4,18 +4,29 @@ import Resolver
 import ModuleLinker
 
 public struct PlayButton: View {
-	@InjectedObject private var audioPlayer: AudioPlayerImpl
+	@InjectedObject private var audioPlayer: VLCAudioPlayer
```

Comment:
What are your thoughts on using a protocol/interface for the AudioPlayer? This way, everywhere that requires an Audio player can use the protocol and have a separate VLCAudioPlayer that implements the AudioPlayer interface. I believe this would make it easier to scale and swap implementations as needed in the future. Additionally, it's probably better coding practice than having the code depend on the concrete implementation. I understand that for now, it may seem like unnecessary work, but it will be helpful down the road, thoughts?